src/routes/acolhido.js

```diff
@@ -1,12 +1,12 @@
-const acolhidos = (acolhido) => (req, res) => {
-    return acolhido
-        .find({
-            where: { id: req.params.id }
-        })
-        .then(acolhidos => {    
-            res.render('pages/info',  { acolhidos : acolhidos })
-        })
-        .catch(console.log)  
-}
+// const acolhido = (acolhido) => (req, res) => {
```

Comment:
Arquivo inteiro de código comentado? 
![judging you](https://media.giphy.com/media/9ZYw9D5K5RpF6/source.gif)